hs_file_types/urls.py

```diff
@@ -22,6 +22,21 @@
         views.remove_aggregation,
         name="remove_aggregation"),
 
+    url(r'^_internal/(?P<resource_id>[0-9a-f]+)/'
+        r'(?P<hs_file_type>[A-z]+)/(?P<file_type_id>[0-9]+)/delete-aggregation/$',
+        views.delete_aggregation,
+        name="delete_aggregation"),
+
+    url(r'^_internal/(?P<resource_id>[0-9a-f]+)/'
+        r'(?P<hs_file_type>[A-z]+)/(?P<file_type_id>[0-9]+)/move-aggregation/(?P<tgt_path>.+)$',
+        views.move_aggregation,
+        name="move_aggregation"),
+
+    url(r'^_internal/(?P<resource_id>[0-9a-f]+)/'
+        r'(?P<hs_file_type>[A-z]+)/(?P<file_type_id>[0-9]+)/move-aggregation/$',
+        views.move_aggregation,
+        name="move_aggregation"),
```

Comment:
There might be a way to consolidate the two url mappings for ```move_aggregation``` above into one url mapping?

Reply:
I'm all ears if you have any ideas.  I did it because we needed a way to map the tgt_path to the root of the resource.